proto/src/ground/v1beta1/survey.proto

```diff
@@ -85,6 +85,24 @@ message Survey {
 
   // Defines the survey's current state.
   State state = 7;
+
+  // Defines the survey's current visibility.
+  enum Visibility {
+    // Default value when visibility is missing.
+    VISIBILITY_UNSPECIFIED = 0;
+
+    // Only users in shared list can contribute.
```

Comment:
`// Only users with the `DATA_COLLECTOR role in the survey ACLs can contribute.`